source/includes/documentation/_material_ui_elements.md

```diff
@@ -2896,17 +2907,23 @@ __The Link__ component allows you to easily customize anchor elements with your
 Here is an example with provided MaterialUI v4.12.3 code:
 
 ```html
-<a class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary" href="#link1">Link</a>
-<a class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit" href="#link2">color="inherit"</a>
+<a class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary" href="#">Link</a>
+<a class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit" href="#">color="inherit"</a>
 ```
 
 Available methods in Java JDI Light:
 
-|Method | Description | Return type
-| --- | --- | --- 
-**is()** | Returns object for work with assertions | LinkAssert
-**isUnderlined()** | Checks that link is underlined | boolean
-**isNotUnderlined()** | Checks that link is not underlined| boolean
+|Method | Description                                                        | Return type
+| --- |--------------------------------------------------------------------| --- 
+**is()** | Returns object for work with assertions                            | LinkAssert
+**isUnderlined()** | Checks that link is underlined                                     | boolean
+**isNotUnderlined()** | Checks that link is not underlined                                 | boolean
+**assertThat()** | Returns object for work with assertions                            | LinkAssert
+**has()** | Returns object for work with assertions                            | LinkAssert
+**waitFor()** | Returns object for work with assertions                            | LinkAssert
+**waitFor(int)** | Waits int sec. and returns object for work with assertions         | LinkAssert
+**shouldBe()** | Returns object for work with assertions                            | LinkAssert
+**verify()** | Set soft assert type, then returns object for work with assertions | LinkAssert
```

Comment:
you can remove all methods from `assertThat()` to `verify()`
why? because _actually all described classes_ are implementing `HasAssert<A>` interface. it's just that `Link` class explicitly overrides methods